Sources/BraintreeCore/BTGraphQLHTTP.m

```diff
@@ -63,12 +63,22 @@ - (void)handleRequestCompletion:(NSData *)data
 {
     // Network error
     if (error) {
-        [self callCompletionBlock:completionBlock body:nil response:nil error:error];
+        [self callCompletionBlock:completionBlock body:nil response:(NSHTTPURLResponse *)response error:error];
         return;
     }
```

Comment:
On line 187, this block is duplicated before calling `handleRequestCompletion` so it can be removed from there.

Reply:
Good catch, updated